modules/renter/contractor/contractor.go

```diff
@@ -6,36 +6,22 @@ import (
 	"path/filepath"
 	"sync"
 
-	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/renter/contractor/proto"
```

Comment:
you mean `/renter/proto` instead of `/renter/contractor/proto`? That seems sensible.